src/lib/client.ts

```diff
@@ -81,14 +81,14 @@ function getStoredBlocks(): FarmedBlock[] {
     for (let [num, block] of Object.entries(blocks)) {
       mined.push(block as FarmedBlock)
     }
-  } catch (error) {
+  } catch (error: any) {
```

Comment:
throws an error if you try to access error like error.toString() since error is 'unknown' type by default.

Reply:
Which in turn means that by slapping `any` you create a time bomb that will blow up once the library throws something that doesn't have `.toString()` method on it. Your need to either specify a concrete type (if there are guarantees that nothing else can possibly be thrown out of there) or write code in such a way that can handle any value.

In this particular case just log the variable itself, `.toString()` will be called on it automatically if implemented, otherwise platform-specific behavior will get at least something printed to the console without causing runtime error.

Reply:
Sure I can do that